NZWalksAPI/Data/NZWalksDbContext.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.EntityFrameworkCore;
+using NZWalksAPI.Models.Domain;
+
+namespace NZWalksAPI.Data
+{
+    public class NZWalksDbContext : DbContext 
+    {
+        public NZWalksDbContext(DbContextOptions dbContextOption) : base(dbContextOption)
+            
+        {
+            
+        }
+        
```

Comment:
Add some more deatiils what you are doing add some comment so that other dev should be undrestand.